File: app/src/main/java/com/damianmichalak/shopping_list/model/Product.java
package com.damianmichalak.shopping_list.model;


public class Product {

    private String name;
    private String info;

    public Product(String name, String info) {
        this.name = name;
        this.info = info;
    }

    public String getInfo() {
        return info;
    }

    public String getName() {

        return name;
    }
}

File: app/src/main/java/com/damianmichalak/shopping_list/model/apiModels/Product.kt
package com.damianmichalak.shopping_list.model.apiModels

data class Product(val id: String = "",
                   val name: String = "",
                   val dateAdded: Long = -1,
                   val datePurchased: Long = -1) {

    fun purchased(): Product {
        return Product(id, name, dateAdded, System.currentTimeMillis())
    }

    fun withId(key: String): Product {
        return Product(key, name, dateAdded, datePurchased)
    }

}
